plugins/Diagnostics/Diagnostic/DiagnosticResult.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+/**
+ * The result of a diagnostic.
```

Comment:
Are there any conventions users of this class should know about? Ie, should users use HTML in their messages if they want (and if they do, should they use <p> elements or are <br/> elements preferred?)? Are the label + comment used in ways that users of this class should be aware of?